### ISSUE TITLE
Cycling windows floods log with assertion failures

### ISSUE THREAD

**Issue by jessestricker** [elementary/gala#1402]
### What Happened?

When cycling windows the system log gets flooded with warnings about two failed assertions in `clutter`.
You can see the warnings below, they get repeated constantly until I release Alt+Tab and switch to the next window and back again.

I run the Pantheon desktop on the latest NixOS-unstable and I've observed this behavior when switching between Firefox and VS Codium.

### Steps to Reproduce

0. Run `journalctl --follow`
1. Open firefox
2. Open codium
4. Press and hold Alt+Tab
5. Observe warning flood

### Expected Behavior

no warnings

### OS Version

Other Linux

### Software Version

Latest release (I have run all updates)

### Log Output

```shell
clutter_paint_volume_set_width: assertion 'width >= 0.0f' failed
clutter_paint_volume_set_height: assertion 'height >= 0.0f' failed
...
```

**Comment by bobby285271** [comment]
This is also reported in https://github.com/NixOS/nixpkgs/issues/159981 and I have no idea yet :crying_cat_face: For context NixOS is shipping mutter 3.38.6 with [these patches](https://github.com/NixOS/nixpkgs/tree/master/pkgs/desktops/gnome/core/mutter/3.38) carried.

**Comment by lenemter** [comment]
@jessestricker @bobby285271 can you still reproduce this issue in your distributions?

**Comment by bobby285271** [comment]
I am no longer able to reproduce this. Bisect suggests this is likely fixed by https://github.com/elementary/gala/pull/1417. I am shipping that patch in NixOS right now. Asked in downstream [issue](https://github.com/NixOS/nixpkgs/issues/159981) to confirm.

https://github.com/NixOS/nixpkgs/blob/193d10b/pkgs/desktops/pantheon/desktop/gala/default.nix#L51

**Comment by lenemter** [comment]
Yes, it was fixed in #1417. I can reproduce the bug by reverting it. Thanks @bobby285271 :heart:

**Comment by jessestricker** [comment]
I no longer use the Pantheon desktop, so I can't reproduce the bug, sorry. Glad to know it's fixed though. Regarding the distro, it's also NixOS, so the setup should be similar to @bobby285271's.
